backgroundchanger/__main__.py

```diff
@@ -42,7 +50,16 @@ def do_wal(photo, do_colors=True):
     utils.reload_gala()
 
 def main():
-    parsed = setup()
+    try:
+        start_log()
+        create_config_folders()
+        create_config_file()
+    except Exception as error:
+        logging.critical(error)
```

Comment:
@rugglcon I am not sure how can I do that, but I think that should be that, right?

Reply:
You should be able to do `logger.exception(error)`, and that will log the entire `Exception` object, including the message and stack trace.

Reply:
I see, thanks!